source/css/scss-common/base/_global-classes.scss

```diff
@@ -476,6 +476,10 @@ hr {
 
     .a-visibleWhenCompleted {
       display: inline-block;
+
+      @include media-breakpoint-up( sm ) {
+        display: inline-block !important;
+      }
```

Comment:
have you also tested a-visibleWhenDeleted, may be same problem exists with that also?

Reply:
The "deleted" is showing. The CSS also looks correct.

![image](https://user-images.githubusercontent.com/9006266/40905561-a57cc3d0-67de-11e8-8836-0de5e6027f83.png)

Reply:
Perfect